### PR TITLE
Bumping NL, Translate to beta

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: google-cloud-language/pom.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>google-cloud-language</artifactId>
+  <version>${beta.version}</version>
```
Comment (vam-google) [inline google-cloud-language/pom.xml]: Should this change be also reflected in google-cloud/pom.xml? Currently it links to ${project.version} for google-cloud-language.

File: google-cloud-translate/pom.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>google-cloud-translate</artifactId>
+  <version>${beta.version}</version>
```
Comment (vam-google) [inline google-cloud-translate/pom.xml]: Similar to the previous case: google-cloud/pom.xml currently it links to ${project.version} for google-cloud-translate.

Comment (garrettjonesgoogle) [inline google-cloud-translate/pom.xml]: yes! good catch.

**Comment by garrettjonesgoogle** [comment]
Readme issues filed:

https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1849
https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1851

And for that matter, we are missing Vision too:

https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1850

**Comment by garrettjonesgoogle** [comment]
Only 1 build left - merging now since this is just a version change and won't be affected by platform issues.

**Pull request merged**